examples/burial-form/PlotAllowance.jsx

```diff
@@ -28,7 +28,7 @@ export default function PlotAllowance(props) {
               ]
             }
           />
-          { formikContext?.values?.federalCemetery === "false" && (
```

Comment:
this should be formikContext?.values?.federalCemetery === false

Reply:
I have added '!' which will is identical to what you're suggesting.